snyk-monitor/README.md

```diff
@@ -98,6 +98,8 @@ helm upgrade --install snyk-monitor snyk-charts/snyk-monitor --namespace snyk-mo
 To better organise the data scanned inside your cluster, the monitor requires a cluster name to be set.
 Replace the value of `clusterName` with the name of your cluster.
 
+**Please note that we do not allow `/` in cluster names.**
```

Comment:
Is this because the issue of the Zendesk ticket? Maybe give a bit more detail saying "if using '/' in  cluster names would result in fail to install the snyk-monitor" etc?